examples/seismic/acoustic/operators.py

```diff
@@ -46,15 +46,11 @@ def ForwardOperator(model, source, receiver, time_order=2, space_order=4,
     eqn = [Eq(u.forward, stencil)]
 
     # Construct expression to inject source values
-    # Note that src and field terms have differing time indices:
-    #   src[time, ...] - always accesses the "unrolled" time index
-    #   u[ti + 1, ...] - accesses the forward stencil value
-    ti = u.indices[0]
-    src_term = src.inject(field=u, u_t=ti + 1, offset=model.nbpml,
-                          expr=src * dt**2 / m, p_t=time)
+    src_term = src.inject(field=u.forward, expr=src * dt**2 / m,
```

Comment:
Yes, but now the feature is unused and untested, and hence assumed to be broken. I really don't get your apprehension here - it's a simple `subs` that can easily added back in by hand for special cases without having a dedicated API in core abstractions, eg. `src.inject(field=u.subs(t, ti + 2), ...)`. If those special cases become frequent, it's also easy to put it back in - but until that time it's just API clutter.